.github/workflows/release-automated.yml

```diff
@@ -31,25 +26,47 @@ jobs:
           key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
           restore-keys: |
             ${{ runner.os }}-gem-
-      - name: Install Bundle
+      - name: Submodules and Bundle Install
         run: |
-          bundle config path vendor/bundle
+          git submodule update --init --recursive
+          sudo gem install bundler
+          bundle config set path 'vendor/bundle'
           bundle install
-      - name: Create Jazzy Docs
-        run: |
-          ./Scripts/jazzy.sh
+      - run: npm ci
+      - run: npx semantic-release
         env:
-          DEVELOPER_DIR: ${{ env.CI_XCODE_13 }}
-      - name: Deploy Jazzy Docs
-        uses: peaceiris/actions-gh-pages@v3
+          GH_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Determine tag on current commit
+        id: tag
+        run: echo "::set-output name=current_tag::$(git describe --tags --abbrev=0 --exact-match || echo '')"
+
+  cocoapods:
+    needs: release
+    if: needs.release.outputs.current_tag != ''
+    runs-on: macos-11
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs
+          ref: ${{ needs.release.outputs.current_tag }}
+      - name: CocoaPods
+        run: set -o pipefail && env NSUnbufferedIO=YES pod lib lint --allow-warnings --verbose
```

Comment:
You can remove this line to save half the time in the pods build. The next line lints and won't deploy if the build fails.

Reply:
ok, let's remove this once we know the original build process with auto-release worked fine.